<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>curve</class>
 <widget class="QDialog" name="curve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>6050-curve</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="Plotter" name="plotwindow">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(76, 76, 76);</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="port">
         <property name="text">
          <string>串口号：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="portComboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="baud">
         <property name="text">
          <string>波特率：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="baudComboBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="parity">
         <property name="text">
          <string>校验位：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="parityComboBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="databits">
         <property name="text">
          <string>数据位：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="databitsComboBox"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="stopbits">
         <property name="text">
          <string>停止位：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="stopbitsComboBox"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="curvename">
         <property name="text">
          <string>曲线名：</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="curveNameComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>&amp;start</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>292</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header>plotter.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
